### DISCUSSION TITLE
Scrummage.py  function() argument 'code' must be code, not str

### DISCUSSION THREAD

**Discussion by The-July-Plot** [matamorphosis/Scrummage#47] [General]
Post-install when running;
~/Scrummage/app$ python3 Scrummage.py

The following error occurs;
function() argument 'code' must be code, not str

###
Debian 11
Python 3.9.9

**Comment by matamorphosis** [comment]
Hi @The-July-Plot,

I have confirmed the latest code runs fine, both through pulling the latest build of the docker image as well as a running a manual installation I have of Scrummage, running on Kali Linux, also with Python 3.9.9. Thus, it doesn't appear to be an issue with the code when Scrummage.

If you could please provide screenshots and further details of the described events as I required further contextual information to help understand your issue. Are you saying this happens as soon as you run `python3 Scrummage.py`

I can only suggest the following based on the info provided:

1. If you are certain and confident you want to proceed with the manual installation, please perform a fresh install following the [installation](https://github.com/matamorphosis/Scrummage/wiki/Installation) guide closely.
2. The simplest solution in my opinion is to download and run the pre-built docker image, with steps shown [here](https://github.com/matamorphosis/Scrummage/wiki/Installation-using-Docker#option-1-using-the-package-link).
3. While I highly doubt using Debian would be a problem, just an FYI that Scrummage was built on Ubuntu 18.04, and the docker image now runs on Ubuntu 20.04.

Let me know how you go.

**Comment by matamorphosis** [comment]
@The-July-Plot as I haven't heard back from you, I assume this issue has either been fixed, or you haven't gotten around to it yet. As there is no confirmed bug, I am going to transfer this issue to a discussion. I can always reopen it as an issue if we confirm one.
